src/main/java/coordinate/domain/Triangle.java

```diff
@@ -0,0 +1,66 @@
+package coordinate.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Triangle implements Figure, ResultPrintable {
+    private static final String RESULT_FORMAT = "삼각형 넓이는 %.1f";
+
+    private final Points points;
+
+    private Triangle(Points points) {
+        this.points = points;
+    }
+
+    public static Triangle create(Points points) {
+        return new Triangle(points);
+    }
+
+    private double getResult(List<Double> distances, double sum, double result) {
+        for (Double distance : distances) {
+            result *= sum - distance;
+        }
+        return result;
+    }
+
+    private double getDistancesSum(List<Double> distances, double sum) {
+        for (Double pointDistance : distances) {
```

Comment:
```suggestion
    distances.stream()
                .mapToDouble(Double::doubleValue)
                .sum();
```
스트림을 쓰면 코드가 더 간결해집니다.
메서드를 없앨 수도 있겠네요